ballerina/types.bal

```diff
@@ -52,6 +52,15 @@ public type CorsConfig record {|
     *http:CorsConfig;
 |};
 
+# Represent GraphiQL client configurations
+#
+# + enable - Status of the client
+# + path - Path for the client
+public type GraphiQL record {|
+    boolean enable;
```

Comment:
Since the `graphiql` is an optional field in Service Configuration record, do we need to add this default value here?

Reply:
Adding it improves the readability right ? By looking at the config one can explicitly know it is not enabled.

Reply:
> Since the `graphiql` is an optional field in Service Configuration record, do we need to add this default value here?

Adding to what @shafreenAnfar mentioned, I think since we have default values for each field, it does not need to be an optional field either. We can make it a required field but disabled it by default.